src/common/xo/index.js

```diff
@@ -1635,6 +1635,12 @@ export const deleteRemote = remote => (
   )
 )
 
+export const deleteRemotes = remotes => (
+  map(remotes, id => _call('remote.delete', { id: resolveId(id) })::tap(
```

Comment:
If it's not assumed that `remotes` is an array of IDs, use "remote" as the param name.